aiozipkin/sampler.py

```diff
@@ -1,9 +1,18 @@
+import abc
 from random import Random
 
 from .mypy_types import OptInt
 
 
-class Sampler:
+class SamplerABC(abc.ABC):
+
+    @abc.abstractmethod
+    def is_sampled(self, trace_id: str) -> bool:
```

Comment:
can you please put `# pragma: no cover` here to disable coverage on this branch?

Reply:
done